README.md

```diff
@@ -243,9 +243,36 @@ This render prop wraps its return value in a list item (`<li>`) and a `<button>`
 | ----------- | ------- | ------------------------------------------------------------------------------------------------------------------------- |
 | active      | Boolean | Whether the current pagination item being iterated over corresponds to the active gallery item.                           |
 | activeIndex | Number  | The index of the currently active gallery item.                                                                           |
-| i           | Number  | The index of the current pagination item being iterated over.                                                             |
+| index       | Number  | The index of the current pagination item being iterated over.                                  |
 | item        | Any     | The current pagination item being iterated over, as defined by the Array fed to the `<Gallery>` component's `items` prop. |
 
+**Example usage of `children` render props**
```

Comment:
Now that we've settled on using the render prop, let's give these docs a quick once-over to make sure we're not referencing the `children` usage.

Reply:
I thought I got rid of them all. Good catch!